browser/app/js/buckets/Policy.js

```diff
@@ -14,65 +14,47 @@
  * limitations under the License.
  */
 
-import { READ_ONLY, WRITE_ONLY, READ_WRITE, NONE } from '../constants'
```

Comment:
Relatively minor, but we can remove the `READ_ONLY, WRITE_ONLY, READ_WRITE` imports as it looks like we aren't using them anymore